### PR TITLE
feat: Homepage v2

### PR DESCRIPTION
**Pull request by DiogoSoaress**
Implementation of the Homepage v2
We're missing some of the animations/assets that are still being developed and can be integrated later.

### Figma

https://www.figma.com/file/qtXXlwdslAze6yNbLLzIRa/Website-redesign-2.0?type=design&node-id=10887-5289&mode=design&t=nzSfAMQi1xtHSLOb-0

### CODE REVIEW

File: src/components/Home/BigIconsCardGrid/index.tsx [hunk 1]
```diff
@@ -10,35 +10,38 @@ import Link from 'next/link'
...
-                </div>
-              </Grid>
-            ))}
-        </Grid>
-      </Container>
-    </div>
+    <Container className={layoutCss.containerMedium}>
+      <Grid container spacing={{ xs: '30px', xl: '50px' }}>
+        {items &&
+          items.map((item, index) => (
```
Comment (iamacook) [inline src/components/Home/BigIconsCardGrid/index.tsx:16]:
```suggestion
        {items?.map((item, index) => (
```

File: src/components/Home/HeroVideo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import css from '@/components/Home/HeroVideo/styles.module.css'
+import type { HeroVideoBlock } from '@/components/Home/types'
+
+const HeroVideo = ({ titleLines: title }: HeroVideoBlock) => {
+  return (
+    <div className={css.container}>
+      <div className={css.wrapper}>
+        <div>
+          <video autoPlay muted playsInline loop className={css.video}>
+            <source src="/videos/Home/safe-2024-hero.mp4" type="video/mp4" />
```
Comment (iamacook) [inline src/components/Home/HeroVideo/index.tsx:11]: I'd suggest moving this to a constant.

File: src/components/Home/HeroVideo/index.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          </video>
+          <div className={css.filter} />
+        </div>
+
+        <Grid container className={css.content} width="100%" height="100%">
+          <Grid item>
+            <Typography variant="h1" className={css.title}>
+              {title[0]}
+              <span className={css.hl}>{title[1]}</span>
```
Comment (iamacook) [inline src/components/Home/HeroVideo/index.tsx:20]: It's a bit odd passing these in an array. Can they be in named properties?

**Review by iamacook — COMMENTED** [review]
I only have a couple comments but I think it'd be beneficial having @usame-algan take a quick look as well.

File: src/components/Home/HeroVideo/index.tsx (hunk 3 above)
Comment (DiogoSoaress) [reply src/components/Home/HeroVideo/index.tsx:20]: Yes, I can change it.

File: public/images/Home/Security/ackee-logo.png
Comment (usame-algan) [inline public/images/Home/Security/ackee-logo.png:1]: This and the other logos are a bit blurry on retina screens. Adding it in 2x might help a bit:
Screenshot 2024-04-19 at 10 18 28

**Review by usame-algan — APPROVED** [review]

**Pull request merged**